src/server/controller/cycleData/persistNodeValue/getDependants.ts

```diff
@@ -0,0 +1,52 @@
+import { CountryIso } from '@meta/area'
+import { Assessment, Cycle, TableNames } from '@meta/assessment'
+
+import { AreaController } from '@server/controller/area'
+import { isODPVariable } from '@server/controller/cycleData/originalDataPointVariables'
+import { OriginalDataPointRepository } from '@server/repository/assessmentCycle/originalDataPoint'
+
+type Props = {
+  assessment: Assessment
+  countryIso: CountryIso
+  cycle: Cycle
+  tableName: string
+  variableName: string
+  colName: string
+  isODP?: boolean
+}
+
+export const isODPCell = async (props: Props) => {
+  const { colName, tableName } = props
+  const country = await AreaController.getCountry(props)
+  const odpYears = await OriginalDataPointRepository.getReservedYears(props)
+  if (![TableNames.forestCharacteristics, TableNames.extentOfForest].includes(tableName as TableNames)) return false
+  return country.props.forestCharacteristics.useOriginalDataPoint && odpYears.map(String).includes(colName)
```

Comment:
this might be wrong in case of table 1a. 1a has odps even when `country.props.forestCharacteristics.useOriginalDataPoint` is false.
correct check should be
```typescript
const useOriginalDataPoint = tableName === TableNames.extentOfForest || country.props.forestCharacteristics.useOriginalDataPoint
return useOriginalDataPoint && odpYears.map(String).includes(colName)
```